tests/data_result/module_130_140/views/sale_order.xml

```diff
@@ -0,0 +1,4 @@
+<odoo>
+    <record id="act_window_example" model="ir.actions.act_window"><field name="name">Example</field><field name="res_model">account.move.line</field><field name="binding_model">account.move</field><field name="view_mode">tree</field></record>
+    <record id="report_example" model="ir.actions.report"><field name="report_name">Name</field><field name="name">String</field></record>
```

Comment:
It's strange that the test is passed after adding pretty_print

Reply:
I hoped for a better XML too (didn't know attribute `pretty_print`), but it only added a blank line

Reply:
Something like this, then?

```
f = open('doc.xml', 'w')
f.write(etree.tostring(root, pretty_print=True))
f.close()

```
https://stackoverflow.com/questions/5086922/python-pretty-xml-printer-with-lxml

Reply:
Thanks for the hint; I couldn't make the `tostring` approach work as I expected so I used the accepted answer of your link's question.
The result is now way better, could you please check?

Reply:
Yes, looks good. Thank you